plugins/dev-tools/package-lock.json

```diff
@@ -6,7 +6,7 @@
 	"packages": {
 		"": {
 			"name": "@blockly/dev-tools",
-			"version": "2.5.5",
```

Comment:
Shouldn't this be a minor (rather than patch) version bump?

Reply:
OK, per discussion (see #861) this is just recording the current version of the package, as bumped by the previous release.